### PR TITLE
Add fingerprinting text and clean up application section flow

### LINKED ISSUE #34: Fingerprinting
A more thorough treatment of this subject might be helpful in [Section 4](https://intarchboard.github.io/draft-obliviousness/draft-kpw-iab-privacy-partitioning.html#section-4).  Fingerprinting exploits characteristics of a client that are innate and therefore difficult to remove.  This includes details of implementation of protocols in hardware and software, and network location.  This manifests in patterns that can, when combined, produce a value that can be used to re-identify clients across contexts.

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #34

### CODE REVIEW

File: draft-iab-privacy-partitioning.md [hunk 1]
```diff
@@ -446,29 +446,55 @@ about individual client data.
...
+User data can itself be user-identifying, in which case it should be treated as an identifier.
 For example, Oblivious DoH and Oblivious HTTP partition the client IP address and client request data into
 separate contexts, thereby ensuring that no entity beyond the client can observe both. Collusion across contexts
 could reverse this partitioning, but can also promote non-user-identifying information to user-identifying.
 For example, in CONNECT proxy systems that use QUIC, the QUIC connection ID is inherently non-user-identifying
 since it is generated randomly ({{?QUIC=RFC9000, Section 5.1}}). However, if combined with another context that has user-identifying
 information such as the client IP address, the QUIC connection ID can become user-identifying information.
 
-This partitioning process can be applied incorrectly or incompletely. Contexts may contain
+Some information is innate to client user-agents, including includes details of implementation of
```
Comment (mirjak) [inline draft-iab-privacy-partitioning.md:472]:
```suggestion
Some information is innate to client user-agents, including details of implementation of
```

File: draft-iab-privacy-partitioning.md [hunk 2]
```diff
@@ -446,29 +446,55 @@ about individual client data.
...
+## Incorrect or Incomplete Partitioning
+
+Privacy partitioning can be applied incorrectly or incompletely. Contexts may contain
 more user-identifying information than desired, or some information in a context may be more user-identifying
-than intended. Moreover, splitting user-identifying information over multiple contexts has to be done with care, as creating more contexts can increase the number of entities that need to be trusted to not collude.
+than intended. Moreover, splitting user-identifying information over multiple contexts has to be done
+with care, as creating more contexts can increase the number of entities that need to be trusted to not collude.
 Nevertheless, partitions can help improve the client's privacy posture when applied carefully.
 
+## Incorrect or Incomplete Partitioning
```
Comment (mirjak) [inline draft-iab-privacy-partitioning.md:487]:
```suggestion
```
Do you have this heading twice or am I not able to read the diff correctly?

File: draft-iab-privacy-partitioning.md [hunk 3]
```diff
@@ -446,29 +446,55 @@ about individual client data.
...
 
-The most impactful types of information to partition are (a) user identity or identities (such as an account name or IP address) that can be linked and (b) user data (such as the content a user is accessing), which can be often sensitive when combined with user identity. Note that user data can itself be user-identifying, in which case it should be treated as an identifier.
+The most impactful types of information to partition are (a) user-identifying information,
+such as user identity or identities (including account names or IP addresses) that can be
+linked and (b) non-user-identifying information, or user data (including the
+content a user is accessing), which can be often sensitive when combined with user identity.
+
+In this section, we discuss considerations for partitioning these types of information.
+
+## Partitioning Considerations
```
Comment (mirjak) [inline draft-iab-privacy-partitioning.md:462]: this heading is not saying much. Would we be able to find a more meaningful heading for this part?

**Review by mirjak — CHANGES_REQUESTED** [review]

File: draft-iab-privacy-partitioning.md (hunk 2 above)
Comment (tfpauly) [reply draft-iab-privacy-partitioning.md:487]: I also see a duplicate heading. @chris-wood What was the intention?

File: draft-iab-privacy-partitioning.md [hunk 4]
```diff
@@ -446,29 +446,55 @@ about individual client data.
...
+1. Identify the types of information used or exposed in a system or protocol, some
+of which can be used to identify a user or correlate to other contexts.
+1.  Partition data to minimize the amount of user-identifying or correlatable
+information in any given context to only include what is necessary for that
+context, and prevent sharing of data across contexts wherever possible.
 
-The most impactful types of information to partition are (a) user identity or identities (such as an account name or IP address) that can be linked and (b) user data (such as the content a user is accessing), which can be often sensitive when combined with user identity. Note that user data can itself be user-identifying, in which case it should be treated as an identifier.
+The most impactful types of information to partition are (a) user-identifying information,
+such as user identity or identities (including account names or IP addresses) that can be
+linked and (b) non-user-identifying information, or user data (including the
```
Comment (tfpauly) [inline draft-iab-privacy-partitioning.md:457]: I think I liked the old phrasing here better, of just calling the second chunk the user data. Maybe user-generated or user-selected data?

File: draft-iab-privacy-partitioning.md (hunk 3 above)
Comment (chris-wood) [reply draft-iab-privacy-partitioning.md:462]: Probably, but it's not obvious to me what that would be.

File: draft-iab-privacy-partitioning.md (hunk 4 above)
Comment (tfpauly) [reply draft-iab-privacy-partitioning.md:457]: It sounds odd to have that be “or user data”. Perhaps we can say “non-user-identifying content the user is accessing or generating”?

File: draft-iab-privacy-partitioning.md (hunk 3 above)
Comment (tfpauly) [reply draft-iab-privacy-partitioning.md:462]: How about “User-identifying information” or “Fingerprinting”, etc

File: draft-iab-privacy-partitioning.md [hunk 5]
```diff
@@ -446,29 +446,54 @@ about individual client data.
...
+content a user is accessing), which can be often sensitive when combined with user identity.
```
Comment (chris-wood) [inline draft-iab-privacy-partitioning.md:458]:
```suggestion
linked and (b) non-user-identifying information (including content a user
generates or accesses), which can be often sensitive when combined with user identity.
```

File: draft-iab-privacy-partitioning.md (hunk 3 above)
Comment (chris-wood) [reply draft-iab-privacy-partitioning.md:462]: I don't think it's much of an improvement, but that works.

File: draft-iab-privacy-partitioning.md [hunk 6]
```diff
@@ -446,29 +446,54 @@ about individual client data.
...
 
-The most impactful types of information to partition are (a) user identity or identities (such as an account name or IP address) that can be linked and (b) user data (such as the content a user is accessing), which can be often sensitive when combined with user identity. Note that user data can itself be user-identifying, in which case it should be treated as an identifier.
+The most impactful types of information to partition are (a) user-identifying information,
+such as user identity or identities (including account names or IP addresses) that can be
+linked and (b) non-user-identifying information (including content a user
+generates or accesses), which can be often sensitive when combined with user identity.
+
+In this section, we discuss considerations for partitioning these types of information.
+
+## Partitioning Considerations
```
Comment (chris-wood) [inline draft-iab-privacy-partitioning.md:462]:
```suggestion
## User-Identifying Information
```

**Comment by tfpauly** [comment]
@mirjak what do you think of the text now?

**Pull request merged**